### PR TITLE
Add Uncolored v0.10.2

### PR DESCRIPTION
**Pull request by rogeriopradoj**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/uncolored.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'uncolored' do
+  version '0.10.2'
+  sha256 '7ac0c07fbe3c0b9b1bf575274e590c7070e9b30f46d16744464ae99d4aed9f07'
+
+  # github.com/n457/Uncolored was verified as official when first introduced to the cask
+  url  "https://github.com/n457/Uncolored/releases/download/v.#{version}/Uncolored-v.#{version}-osx-x64.dmg"
```
Comment (vitorgalvao) [inline Casks/uncolored.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by rogeriopradoj** [comment]
Whow, thanks @miccal , sorry bothering you (about the comma). @vitorgalvao I think it's ok by now?

**Comment by rogeriopradoj** [comment]
Ops, no... @miccal . Do you prefer I update it on my machine and test before sending again? (sorry about that again (I've include appcast via github webpage).

**Review by vitorgalvao — APPROVED** [review]
Everything seems fine. We can squash when merging.

**Pull request merged**